src/App.js

```diff
@@ -1,23 +1,337 @@
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+    <div>
+      <header className="header">
+        <section>
+          <h1 className='header-title'>The Book Spot</h1>
+        <Form
+        handleSubmitFunction={handleSubmit}
+        handleCounter={changeCounter}
+        />
+        </section>
+        <button onClick={toggleCart} className='book-store'><FontAwesomeIcon icon={ faBookReader } />
+        {
+          fireStorage.length > 0 && !cartView ? 
+          <p className='notification'>{fireStorage.length}</p>
+          : null
+        }
+        </button>
+        {cartView &&
+          <section className='cartMenu'>
+                  {
+                    fireStorage.map((individualStoredBook) => {
+          
+                      return (
+                        <BookCart 
+                        title={individualStoredBook.bookInfo.title}
+                        imgSrc={individualStoredBook.bookInfo.src}
+                        id={individualStoredBook.key}
+                        />
+                      )
+                    })
+                  }
+          </section>
+        }
       </header>
+      <main>
+        <div className='wrapper'>
+          <section className='book-collection'>
+              {!isHidden && <TailSpin 
+              height={100}
+              width={100} 
+              color='black' 
+              ariaLabel='loading'
+              className="loading-icon" 
+              />}
+
+                {
+                  error && <h2 className='loading-icon'>Sorry the network seems to be down, please try again soon</h2>
+                }
+
+                  { !view &&
+                  <>
+                  <FilterBooks
+                  term={searchTerm}
+                  getMoreBooks={handleSubmit}
+                  bookArr={books}
+                  filteredBooks={filteredBooks}
+                  />
+                  <ul className='book-list'>              
+                    {books.map((bookObj) => {
+                        return (
+                          <li className="flex-list-item" key={bookObj.id}>
+                            <button className='fav-button' onClick={handleAddBook} value={bookObj.id}>{bookObj.clicked ?<FontAwesomeIcon  icon={ heart } /> 
+                              :  <FontAwesomeIcon icon={ farHeart } />
+                          
+                          }</button>
+
+                            <button className='article-modal' onClick={() => {changeView(bookObj)}}>
+                              <Book
+                              // error handling for if bookObj.volumeInfo.imageLinks is not valid
```

Comment:
Nice. Error handling is super important. So is breakfast. Both of them should happen during every project. Sometimes second breakfast.